purenix/src/Nix/Print.hs

```diff
@@ -99,6 +100,7 @@ exprPrec (Bin op _ _) = opPrec op
     opPrec Update = 5
 exprPrec Abs {} = 0
 exprPrec Let {} = 0
+exprPrec Raw {} = 0
```

Comment:
I wasn't quite sure what to put here.

My goal was to make it so that `Raw` nodes would always be rendered with parenthesis around them, but I couldn't quite get this working.